notebooks/finetuning/Command R finetuning sagemaker.ipynb

```diff
@@ -0,0 +1,472 @@
+    "\n",
+    "> **Note**: This is a reference notebook and it cannot run unless you make changes suggested in the notebook.\n",
+    "\n",
+    "## Pre-requisites:\n",
+    "1. **Note: This notebook contains elements which render correctly in Jupyter interface. Open this notebook from an Amazon SageMaker Notebook Instance or Amazon SageMaker Studio.**\n",
+    "1. Ensure that IAM role used has **AmazonSageMakerFullAccess**\n",
+    "1. To deploy this ML model successfully, ensure that:\n",
+    "    1. Either your IAM role has these three permissions and you have authority to make AWS Marketplace subscriptions in the AWS account used: \n",
+    "        1. **aws-marketplace:ViewSubscriptions**\n",
+    "        1. **aws-marketplace:Unsubscribe**\n",
+    "        1. **aws-marketplace:Subscribe**  \n",
+    "    2. or your AWS account has a subscription to the packages for [Cohere Command R Finetuning](https://aws.amazon.com/marketplace/ai/configuration?productId=1762e582-e7df-47f0-a49f-98e22302a702). If so, skip step: [Subscribe to the finetune algorithm](#1.-Subscribe-to-the-finetune-algorithm)\n",
+    "\n",
+    "## Contents:\n",
+    "1. [Subscribe to the finetune algorithm](#1.-Subscribe-to-the-finetune-algorithm)\n",
+    "2. [Upload data and finetune Command-R Model](#2.-Upload-data-and-finetune-Command-R)\n",
+    "3. [Create an endpoint for inference with the custom model](#3.-Create-an-endpoint-for-inference-with-the-custom-model)\n",
+    "   1. [Create an endpoint](#A.-Create-an-endpoint)\n",
+    "   2. [Perform real-time inference](#B.-Perform-real-time-inference)\n",
+    "4. [Clean-up](#4.-Clean-up)\n",
+    "    1. [Delete the endpoint](#A.-Delete-the-endpoint)\n",
+    "    2. [Unsubscribe to the listing (optional)](#Unsubscribe-to-the-listing-(optional))\n",
+    "    \n",
+    "\n",
+    "## Usage instructions\n",
+    "You can run this notebook one cell at a time (By using Shift+Enter for running a cell)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Subscribe to the finetune algorithm"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To subscribe to the model algorithm:\n",
+    "1. Open the algorithm listing page [Cohere Command R Finetuning](https://aws.amazon.com/marketplace/pp/prodview-2czs5tbao7b7c)\n",
+    "2. On the AWS Marketplace listing, click on the **Continue to Subscribe** button.\n",
+    "3. On the **Subscribe to this software** page, review and click on **\"Accept Offer\"** if you and your organization agrees with EULA, pricing, and support terms. On the \"Configure and launch\" page, make sure ARN displayed in your region match with the ARN in the following cell."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "!pip install cohere\n",
```

Comment:
```suggestion
    "!pip install cohere>=0.5.11\n",
```

Reply:
Should be `!pip install cohere>=5.11.0`?

Reply:
Yes